chain/chunks/src/lib.rs

```diff
@@ -1356,23 +1393,27 @@ impl ShardsManager {
             partial_encoded_chunk.map(|chunk| PartialEncodedChunkV2::from(chunk));
         let header = &partial_encoded_chunk.header;
         let chunk_hash = header.chunk_hash();
-        debug!(
+        let _span = tracing::debug_span!(
             target: "chunks",
+            "process_partial_encoded_chunk",
             ?chunk_hash,
-            height = header.height_created(),
             shard_id = header.shard_id(),
+            height_created = header.height_created(),
+            height_included = header.height_included())
+        .entered();
+        tracing::debug!(
+            target: "chunks",
             parts = ?partial_encoded_chunk.get_inner().parts.iter().map(|p| p.part_ord).collect::<Vec<_>>(),
-            "Process partial encoded chunk",
-        );
+            "Process partial encoded chunk");
         // Verify the partial encoded chunk is valid and worth processing
         // 1.a Leave if we received known chunk
         if let Some(entry) = self.encoded_chunks.get(&chunk_hash) {
             if entry.complete {
                 return Ok(ProcessPartialEncodedChunkResult::Known);
             }
-            debug!(target: "chunks", "{} parts in cache, total needed: {}", entry.parts.len(), self.rs.data_shard_count());
+            debug!(target: "chunks", num_parts_in_cache = entry.parts.len(), total_needed = self.rs.data_shard_count());
         } else {
-            debug!(target: "chunks", "0 parts in cache, total needed: {}", self.rs.data_shard_count());
+            debug!(target: "chunks", total_needed = self.rs.data_shard_count(), "0 parts in cache");
```

Comment:
Nit: I think this would be better off if these two events used the same format (right now they are only usable for reading logs – if you filter by `num_parts_in_cache` you will always get a non-zero value which is going to be potentially misleading)